meta-intel-edison-distro/recipes-core/images/edison-image.bb

```diff
@@ -166,3 +166,7 @@ IMAGE_INSTALL += "battery-voltage"
 IMAGE_INSTALL += "perf"
 
 IMAGE_INSTALL += "sketch-check"
+
+# package management - where to find this?
+#IMAGE_INSTALL += "aptitude"
```

Comment:
It's [recommended](http://www.yoctoproject.org/docs/2.3.2/ref-manual/ref-manual.html#var-IMAGE_INSTALL) to use the `_append` modifier instead of `+=` operator with `IMAGE_INSTALL`. In addition, the package name is `apt`, not `aptitude`.

Reply:
apt is already installed, I was looking for aptitude but can't find a source for that.
+= used here from the original poky-edison.conf, change all to IMAGE_INSTALL_append (and ' ' between " and package name)?

Reply:
I see, so you're after `aptitude` specifically. There indeed seems to be no Yocto recipe available in any of the official layers, so if your really want it, a recipe must be created from scratch.

As for the `_append` thing - yes, I'd suggest correcting this (with the leading space, right). Edison recipes did have some bugs or let's say non-recommended configs, yes :)

Reply:
aptitude is not really high priority, _append think OK I'll fix that